portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,31 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
```

Comment:
These functions are called 'add' but they seem to do replacing instead?

Reply:
Ah I see now. This is adding a fact the first time it's clicked and replacing it the second time?

Perhaps a name like addOrReplaceFact() would make more sense? You can also leave it as is, if you prefer.